integration_tests/mockApis/userPreference.ts

```diff
@@ -85,6 +85,46 @@ export default {
       },
     })
   },
+
+  stubUserPreferenceAllocationDemand: ({
+    pduCode,
+    lduCode,
+    teamCode,
+  }: {
+    pduCode: string
+    lduCode: string
+    teamCode: string
+  }): SuperAgentRequest => {
+    return stubForUserPreference({
+      request: {
+        method: 'GET',
+        urlPattern: `/users/USER1/preferences/allocation-demand`,
+      },
+      response: {
+        status: 200,
+        headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+        jsonBody: {
+          items: [`{"pdu": "${pduCode}", "ldu": "${lduCode}", "team": "${teamCode}"}`],
```

Comment:
this is a bit of a hack. the column in the user preference table it 255 characters long so this json string will always fit https://github.com/ministryofjustice/hmpps-user-preferences/blob/main/src/main/resources/db/migration/V1_0__create_preferences_table.sql#L6